sdk/storage/storage-blob/samples/typescript/azureAdAuth.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ Setup: Enter your Azure Active Directory credentials as described in main()
+*/
+
+import { BlobServiceClient, SharedKeyCredential } from "../../src"; // Change to "@azure/storage-blob" in your package
+import { DefaultAzureCredential } from "@azure/identity";
+
+async function main() {
```

Comment:
>main [](start = 15, length = 4)

Is "Ad" or "Aad"? is this file name `azureAdAuth.ts` correct?

Reply:
I called it "azureAd" to refer to "Azure Active Directory", though I can see how having "aad" explicitly could make things clearer.  I'll ask someone about this and will rename these files after the PR gets merged if necessary.